bindings/wysiwyg-ffi/Cargo.toml

```diff
@@ -17,10 +17,10 @@ crate-type = ["cdylib", "staticlib"]
 # Keep the uniffi version here in sync with the installed version of
 # uniffi-bindgen that is called from
 # ../../examples/example-android/app/build.gradle
-uniffi = "=0.19.2"
-uniffi_macros = "=0.19.2"
+uniffi = "=0.21.0"
```

Comment:
Can you change this value as well on both `.github/workflows/android.yml` and `.github/workflows/ios.yml` ? 

Plus, I suggest adding Android and iOS to make sure platform specific CI works as expected.